test/index.test.js

```diff
@@ -275,17 +275,17 @@ describe('Reporter', function () {
 
                 const request = reporter.Finance.getStatus();
 
-                expect(stub.firstCall.args).to.eql(['/finance/v1', 'Finance.getStatus']);
+                expect(stub.firstCall.args).to.eql(['/finance/v1', 'Finance.getStatus', { account: undefined }]);
             }));
         });
 
         describe('getVendorsAndRegions', function () {
             it('should have correct parameters', sinon.test(function () {
                 const stub = this.stub(reporter, 'fetch');
 
-                const request = reporter.Finance.getVendorsAndRegions();
+                const request = reporter.Finance.getVendorsAndRegions({}, { account: 654321 });
 
-                expect(stub.firstCall.args).to.eql(['/finance/v1', 'Finance.getVendorsAndRegions']);
+                expect(stub.firstCall.args).to.eql(['/finance/v1', 'Finance.getVendorsAndRegions', { account: 654321 }]);
```

Comment:
Was this intended to have a set value `654321`, or should it be set a `undefined`?

Reply:
It should be set since the second argument on `reporter.Finance.getVendorsAndRegions` is `{ account: 654321 }`.